src/components/search/counter-filter.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import { injectIntl } from 'react-intl'
+import { connect } from 'react-redux'
+import { VALUE_TYPE_FLOAT, FILTER_OPERATOR_GREATER_OR_EQUAL } from 'components/search/constants'
+
+class CounterFilter extends Component {
+  constructor(props) {
+    super(props)
+
+    this.defaultValue = this.props.defaultValue ? this.props.defaultValue : 1
+    this.state = {
+      counter: this.defaultValue
+    }
+
+    this.handleOnClickAdd = this.handleOnClickAdd.bind(this)
+    this.handleOnClickSubstract = this.handleOnClickSubstract.bind(this)
```

Comment:
While @franckc can inevitably produce some obscure, ancient reference to "substract" being a word, I recommend that all instances of this be replaced with "subtract". ➖